frontend/src/views/CreatePollView.vue

```diff
@@ -1,7 +1,8 @@
 <script setup lang="ts">
 import { ref } from 'vue';
+import { ethers } from 'ethers';
 
-import { useDAOv1 } from '../contracts';
+import { useDAOv1, useGaslessVoting, useUnwrappedDAOv1, useUnwrappedGaslessVoting } from '../contracts';
```

Comment:
Could you move this to a separate PR please? I would need that for my GSN voting as well and let's merge it separately ;)